### PR TITLE
Updated doc of distributed_ddl_queue

### PR DESCRIPTION
**Pull request by alexon1234**
### Changelog category (leave one):
- Documentation (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

Update the documentation of `system.distributed_ddl_queue` with the latest changes

### CODE REVIEW

File: docs/en/operations/system-tables/distributed_ddl_queue.md [hunk 1]
```diff
@@ -8,17 +8,21 @@ Contains information about [distributed ddl queries (ON CLUSTER clause)](../../s
...
+- `entry_version` ([Nullable(UInt8)](../../sql-reference/data-types/int-uint.md)) - Version of the entry
+- `initiator_host` ([Nullable(String)](../../sql-reference/data-types/string.md)) - Host that initiated the DDL operation
+- `initiator_port` ([Nullable(UInt16)](../../sql-reference/data-types/int-uint.md)) - Port used by the initiator
 - `cluster` ([String](../../sql-reference/data-types/string.md)) — Cluster name.
 - `query` ([String](../../sql-reference/data-types/string.md)) — Query executed.
-- `initiator` ([String](../../sql-reference/data-types/string.md)) — Node that executed the query.
-- `query_start_time` ([DateTime](../../sql-reference/data-types/datetime.md)) — Query start time.
+- `settings` ([Map(String, String)](../../sql-reference/data-types/map.md)) - Settings used in the DDL operation
+- `query_create_time` ([DateTime](../../sql-reference/data-types/datetime.md)) — Query created time.
+- `host` ([String](../../sql-reference/data-types/string.md)) — HostName
```
Comment (tavplubix) [inline docs/en/operations/system-tables/distributed_ddl_queue.md:18]:
```suggestion
- `host` ([String](../../sql-reference/data-types/string.md)) — Hostname
```

File: docs/en/operations/system-tables/distributed_ddl_queue.md [hunk 2]
```diff
@@ -8,17 +8,21 @@ Contains information about [distributed ddl queries (ON CLUSTER clause)](../../s
...
+- `port` ([UInt16](../../sql-reference/data-types/int-uint.md)) — Host Port.
+- `status` ([Enum8](../../sql-reference/data-types/enum.md)) — Status of the query.
+- `exception_code` ([Enum8](../../sql-reference/data-types/enum.md)) — Exception code from [ClickHouse Keeper](../../operations/tips.md#zookeeper).
```
Comment (tavplubix) [inline docs/en/operations/system-tables/distributed_ddl_queue.md:21]:
```suggestion
- `exception_code` ([Enum8](../../sql-reference/data-types/enum.md)) — Exception code.
```

File: docs/en/operations/system-tables/distributed_ddl_queue.md [hunk 3]
```diff
@@ -8,17 +8,21 @@ Contains information about [distributed ddl queries (ON CLUSTER clause)](../../s
...
+- `exception_text` ([Nullable(String)](../../sql-reference/data-types/string.md)) - Exception error
```
Comment (tavplubix) [inline docs/en/operations/system-tables/distributed_ddl_queue.md:22]:
```suggestion
- `exception_text` ([Nullable(String)](../../sql-reference/data-types/string.md)) - Exception message
```

**Review by tavplubix — APPROVED** [review]

**Comment by alexon1234** [comment]
Thanks @tavplubix for the suggestions.

**Comment by robot-ch-test-poll2** [comment]
This is an automated comment for commit 86ee29ada06dfc414581eb2389031fca5b067660 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/54192/86ee29ada06dfc414581eb2389031fca5b067660/ci_running.html)
The overall status of the commit is 🟡 pending

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🟡 pending</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

**Pull request merged**